### DISCUSSION TITLE
Latest Release should be listed as a breaking change

### DISCUSSION THREAD

**Discussion by xela1** [LightwaveSmartHome/homeassistant-lightwave-smart#2] [General]
Hi Ivan

I've installed the latest version on my test system, and unless my experience is incorrect, I believe you should list breaking changes on the latest release. 

I see you've changed certain entities (light switches, socket switches, thermostats, LEDs etc), if anyone has these entities in automations, scripts, dashboards etc, then they will need to replace the existing entities with the new ones

**Comment by ikb42** [comment]
Hi @xela1 
Thanks for the feedback.
Yes, this release deviates quite a bit from the source / original implementation in the way it presents devices / entities to Home Assistant, and indeed from the initial release.
It should be stable in this regard from here on.

When this forked version makes it into HACS as a default integration / repository it will be under a new label "Lightwave Smart custom component" so it wont be an update to the original.
So I'm not sure if breaking changes need to be noted?
Sorry if this has caused you problems.

**Reply by xela1** [reply]
> Sorry if this has caused you problems.

It's not a problem, it's great to see the integration getting some effort (no offense intended to Bryan here, you did some great work), I just think it might be worth warning people that it may require some changes when upgrading, but I appreciate it will be a 'new' integration once it gets in to HACS.

**Reply by ikb42** [reply]
Makes sense, ok will look to add a note to the readme.  Please feel free to do a PR updating the readme if you like.

**Comment by ikb42** [comment]
Added note to readme
